QUICKSTART-VMWARE.md

```diff
@@ -0,0 +1,248 @@
+
+#### API Server
+This is the address (including port) of the control plane.
+You can run the following to get it:
+```
+ kubectl --kubeconfig=<your kubeconfig> cluster-info
+```
+The value you need will be found after `Kubernetes control plane is running at:` and should start with `https://` and end with a port.
+
+#### Cluster DNS IP
+This is the IP address of the DNS pod/service.
+```
+kubectl --kubeconfig=<your kubeconfig> -n kube-system get svc -l k8s-app=kube-dns
+```
+The value you need will be found under `CLUSTER-IP` and is an IP address.
+
+#### Bootstrap token
+Nodes require a token to establish trust between the node and the control plane.
+Tokens must be [used within 24 hours](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-token/), but once the node has booted and registered with the cluster, it isn't used again.
+```
+kubeadm --kubeconfig <your kubeconfig> token create
+```
+
+#### Cluster Certificate
+This is the base64 encoded cluster certificate authority.
+```
+kubectl --kubeconfig=<your kubeconfig> config view --raw -o json | jq -r '.clusters[0].cluster."certificate-authority-data"'
+```
+
+## Configuring Bottlerocket
+
+In order to join Bottlerocket to your cluster, it must be configured via user data.
+There are multiple methods of passing user data to Bottlerocket in VMware; we will demonstrate all of them.
+
+Create a file called `userdata.toml` and populate it with the values you just retrieved.
+```
+[settings.kubernetes]
+api-server = "<value here>"
+cluster-dns-ip = "<value here>"
+bootstrap-token = "<value here>"
+cluster-certificate = "<value here>"
+```
+
+### Accessing your Bottlerocket guest via host containers
+For remote access to your running Bottlerocket VMs, you will need to add additional user data to enable host containers.
+The Bottlerocket VMware images don't include any host containers enabled by default.
+But don't worry!
+You can use our [admin](https://github.com/bottlerocket-os/bottlerocket-admin-container) and/or [control](https://github.com/bottlerocket-os/bottlerocket-control-container) containers, they just need to be configured first.
+
+#### Admin container
+If you would like to use the admin container, you will need to create some base64 encoded user data which will be passed to the container at runtime.
+Details for the required host container user data are covered in the [admin container documentation](https://github.com/bottlerocket-os/bottlerocket-admin-container#authenticating-with-the-admin-container).
+Once you've created the required admin container user data, you can enable the admin container by adding the following to your Bottlerocket user data:
+```
+[settings.host-containers.admin]
+enabled=true
+user-data = "<YOUR USER DATA>"
+```
+
+#### Control container
+Enabling the control container is very similar to the admin container; you will create some base64 encoded user data that will be passed to the container at runtime.
```

Comment:
For this we should give them a copy/paste command that uses the AWS CLI to create the SSM values, then populates or extends user-data.